src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -171,17 +190,19 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
 
     register_cli_argument(scope, 'os_disk_name', help='The name of the new VM OS disk.', arg_group='Storage')
     register_cli_argument(scope, 'os_type', help='Type of OS installed on a custom VHD. Do not use when specifiying an URN or URN alias.', arg_group='Storage', **enum_choice_list(['windows', 'linux']))
-    register_cli_argument(scope, 'storage_account', help='The name to use when creating a new storage account or referencing an existing one. If omitted, an appropriate storage account in the same resource group and location will be used, or a new one will be created.', arg_group='Storage')
+    register_cli_argument(scope, 'storage_account', help="Only applicable when use with '--use-unmanaged-disk'. The name to use when creating a new storage account or referencing an existing one. If omitted, an appropriate storage account in the same resource group and location will be used, or a new one will be created.", arg_group='Storage')
     register_cli_argument(scope, 'storage_caching', help='Storage caching type for the VM OS disk', arg_group='Storage', **enum_choice_list(['ReadWrite', 'ReadOnly']))
-    register_cli_argument(scope, 'storage_sku', help='The storage SKU to use for new storage accounts.', arg_group='Storage', **enum_choice_list(SkuName))
-    register_cli_argument(scope, 'storage_container_name', help='Name of the storage container for the VM OS disk.', arg_group='Storage')
+    register_cli_argument(scope, 'storage_sku', help="The storage SKU to use for new storage accounts. Allowed value: Premium_LRS, Standard_LRS. With use with '--use-unmanaged-disk', 3 more options are available: Standard_GRS, Standard_RAGRS, Standard_ZRS. ", arg_group='Storage')
+    register_cli_argument(scope, 'storage_container_name', help="Only applicable when use with '--use-unmanaged-disk'. Name of the storage container for the VM OS disk.", arg_group='Storage')
 
     register_cli_argument(scope, 'os_publisher', ignore_type)
     register_cli_argument(scope, 'os_offer', ignore_type)
     register_cli_argument(scope, 'os_sku', ignore_type)
     register_cli_argument(scope, 'os_version', ignore_type)
     register_cli_argument(scope, 'storage_profile', ignore_type)
-
+    register_cli_argument(scope, 'use_unmanaged_disk', action='store_true', help='Do not use managed disk to persist VM', arg_group='Storage')
+    register_cli_argument(scope, 'data_disk_sizes_gb', nargs='+', type=int, help='space separated empty managed data disk sizes in GB to create', arg_group='Storage')
+    register_cli_argument(scope, 'image_data_disks', help=argparse.SUPPRESS)
```

Comment:
Please use `ignore_type` instead of `help=argparse.SUPPRESS`. `ignore_type` has additional behavior.